scripts/ingests/21Legg_ingest.py

```diff
@@ -0,0 +1,449 @@
+        publication='Mace13.36').execute()
+    db.Publications.update().where(db.Publications.c.publication == 'Pinf14').values(
+        publication='Pinf14.1931').execute()
+    db.Publications.update().where(db.Publications.c.publication == 'Pinf14a').values(
+        publication='Pinf14.1009').execute()
+    db.Publications.update().where(db.Publications.c.publication == 'Scho10a').values(
+        publication='Scho10.92').execute()
+    db.Publications.update().where(db.Publications.c.publication == 'Scho10b').values(
+        publication='Scho10.8').execute()
+    db.Publications.update().where(db.Publications.c.publication == 'Tinn05a').values(
+        publication='Tinn05.1171').execute()
+    db.Publications.update().where(db.Publications.c.publication == 'Tinn05b').values(
+        publication='Tinn05.2326').execute()
+
+    db.Publications.update().where(db.Publications.c.publication == 'Warr07').values(
+        publication='Warr07.1400').execute()
+    db.Publications.update().where(db.Publications.c.publication == 'Warr07a').values(
+        publication='Warr07.213').execute()
+
+
+def convert_ref_name(reference):
+    ref_dict = {'Burgasser_2000': 'Burg00.57',
+                'Burgasser_2002': 'Burg02.421',
+                'Burgasser_2003': 'Burg03.2487',
+                'Burgasser_2006': 'Burg06.1067',
+                'Burgasser_2004': 'Burg04.2856',
+                'Burgasser_2010b': 'Burg02.421',  # Error in Leggett 2021
+                'Burningham_2010a': 'Burn10.1885',
+                'Burningham_2010b': 'Burn10.1885',
+                'Burningham_2011': 'Burn11.90',
+                'Cushing_2011': 'Cush11.50',
+                'Delorme_2008': 'Delo08.961',
+                'Dupuy_2015': 'Dupu15.102',
+                'Gelino_2011': 'Geli11.57',
+                'Kirkpatrick_2012': 'Kirk12',
+                'Kirkpatrick_2013': 'Kirk13',
+                'Liu_2011': 'Liu_11.108',
+                'Lodieu_2007': 'Lodi07.1423',
+                'Lodieu_2009': 'Lodi09.258',
+                'Lodieu_2012': 'Lodi12.53',
+                'Looper_2007': 'Loop07.1162',
+                'this_work': 'Legg21',
+                'Lucas_2010': 'Luca10',
+                'Luhman_2012': 'Luhm12.135',
+                'Luhman_2014': 'Luhm14.18',
+                'Mace_2013a': 'Mace13.6',
+                'Mace_2013b': 'Mace13.36',
+                'Meisner_2020a': 'Meis20.74',
+                'Meisner_2020b': 'Meis20.123',
+                'Pinfield_2014a': 'Pinf14.1009',
+                'Pinfield_2014b': 'Pinf14.1931',
+                'Pinfield_Gromadzki_2014': 'Pinf14.priv',
+                'Scholz_2010a': 'Scho10.8',
+                'Scholz_2010b': 'Scho10.92',
+                'Tinney_2005': 'Tinn05.2326',
+                'Warren_2007': 'Warr07.1400'
+                }
+
+    # convert reference names
+    first_ref = reference.split()[0]
```

Comment:
this is a split on white space, is that correct?

Reply:
Yes, that's how the leggett table formatted multiple refs in one cell.